### PR TITLE
ClaudiaWeek2 - Created Servlet that is added to the portfolio.

### PR DESCRIPTION
**Pull request by claudiapao10**
Create the servlet that is added to the portfolio when user request an URL.

### CODE REVIEW

**Comment by GoogleyGabriel** [comment]
Usually, you want to specify in the descriptions which files you changed and what you did to them. For the *.xml files, you can do a catch-all, like "ran command _____ to update the version from _____ to _______ to all xml files."

File: portfolio/pom.xml [hunk 1]
```diff
@@ -12,11 +12,12 @@
...
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jetty.version>11.0.0</jetty.version>
+    <jetty.version>9.4.31.v20200723</jetty.version>
 
     <!-- Project-specific properties -->
     <exec.mainClass>com.google.sps.ServerMain</exec.mainClass>
-    <googleCloudProjectId>YOUR_PROJECT_ID_HERE</googleCloudProjectId>
+    <googleCloudProjectId>cmongetorres-sps-spring21
+</googleCloudProjectId>
```
Comment (GoogleyGabriel) [inline portfolio/pom.xml]: This can be in the same line.

File: portfolio/src/main/java/portfolio/src/main/java/PortfolioServer.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns HTML that contains the page view count. */
+@WebServlet("/page-salutation")
+public class PortfolioServer extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
```
Comment (GoogleyGabriel) [inline portfolio/src/main/java/portfolio/src/main/java/PortfolioServer.java:29]: Nit: no new line.

File: portfolio/src/main/webapp/about-me.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        </nav>
+
+        <h1>About Me: Claudia Monge</h1>
+        <div class="about-img"> <img src="images/about-me-pic-resize.jpg" alt=""></div>
+
+
+        <div class="about-main-content">
+            <p><b>Hello!</b> My name is Claudia Monge Torres. Currently and at the moment, I am a full-time computer science
+                student. <br> I originated in the town of <a
+                    href="https://www.google.com/maps/place/Humacao,+Puerto+Rico/@22.620792,-56.4098252,3.09z/data=!4m5!3m4!1s0x8c04a898d87093e3:0xceb5e85b1704d482!8m2!3d18.1551337!4d-65.8203943">Humacao,
```
Comment (GoogleyGabriel) [inline portfolio/src/main/webapp/about-me.html]: Nit: Why the line break?

File: portfolio/src/main/webapp/about-me.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+                    Puerto Rico</a> and continued my studies in Bayamón, where I have been able to have my beginning in university
+                life being a by present in clubs and associations including "Women in Cybersecurity - Bayamón Chapter" and the
```
Comment (GoogleyGabriel) [inline portfolio/src/main/webapp/about-me.html]: "by president" -> "vice-president"

File: portfolio/src/main/webapp/about-me.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+                "Computer Science Student Association". These experiences disciplined me and made me know more about my passions
+                within programming. <br> One topic that I am passionate about is improving the local economy and the growth of small
```
Comment (GoogleyGabriel) [inline portfolio/src/main/webapp/about-me.html]: Usually when you put the `<br>` tag in its own line, and is then followed by a new line. The tag just ensures that the line in fact takes place when the text is rendered into the page.

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -1,9 +1,370 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+
```
Comment (GoogleyGabriel) [inline portfolio/src/main/webapp/style.css]: Nit: unnecessary white lines, but looking good over all!

**Review by GoogleyGabriel — APPROVED** [review]
Looking great! Just some minor formatting stuff (it will be a lot of those until we get to more logic intensive stuff in JS and Java files, so don't worry).

Also, try to keep your PR requests small (Like one for the Java file, and one per HTML+css file).

**Review by sebagabs — APPROVED** [review]
Looking great!

**Review by sebagabs — APPROVED** [review]
Nice, that looks better.

File: portfolio/src/main/java/PortfolioServer.java [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+public class PortfolioServer extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    response.setContentType("text/html;");
+    response.getWriter().println("<h1>hello world</h1>");
+    
+  }
+}
```
Comment (sebagabs) [inline portfolio/src/main/java/PortfolioServer.java:34]: Looking great!

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -358,13 +338,12 @@ margin-left:  100px;
...
 }
 
 .form div{
     width: 500px;
 }
 
 .form button{
-width: 100px;
+    width: 100px;
 }
```
Comment (sebagabs) [inline portfolio/src/main/webapp/style.css:351]: That looks better!

File: portfolio/src/main/webapp/script.js [hunk 9]
```diff
@@ -26,3 +26,11 @@ function addRandomGreeting() {
...
   greetingContainer.innerText = greeting;
 }
+
+async function ShowServerText(){
+    const responseFromServer = await fetch('/page-salutation');
+    const textFromResponse = await responseFromServer.text();
+
+    const textContatiner = document.getElementById('text-container');
+    textContatiner.innerText = textFromResponse;
+}
```
Comment (sebagabs) [inline portfolio/src/main/webapp/script.js:36]: Great idea to add the function here; I added a new file but this seems easier.

File: portfolio/src/main/webapp/about-me.html [hunk 10]
```diff
@@ -86,27 +80,29 @@ <h5 class="card-title">Topics that I like</h5>
...
+            <p class="about-info">From here to the future, I want to be able to experiment in different areas of computer science
+                and choose some to continue my studies. I envision myself during this time being a voice for girls in tech. I want
+                to do a mentoring program to encourage the participation of more girls in the STEM area. In this way, it could help
+                to close the gender gap in these areas. <br>As I mentioned earlier, I would also like to be able to carry out the
+                project that seeks to improve the local economy and the growth of small businesses in Puerto Rico. This is, since a
+                large part of the Puerto Rican economy falls on small and local businesses.<br>On top of that, one of my goals is to be
+                part of the people who empower and inspire other women to be part of the movement to increase the representation of
+                women in STEM. Since there is already knowledge about the gender disparity in STEM, I would like to come up with an
+                initiative or be part of one as this would give girls and women opportunities in the field beforehand.
+            <p>
```
Comment (sebagabs) [inline portfolio/src/main/webapp/about-me.html]: Shouldn't this line be `</p>`? I'm not seeing the closing tag, but if you have it don't worry!

File: portfolio/src/main/webapp/gallery.html [hunk 11]
```diff
@@ -45,18 +45,18 @@ <h1 class="gallery-header">CLAUDIA'S GALLERY</h1>
...
-            <img src="images/sky.jpg" alt="">
+            <img src="images/igor.jpg" alt="Igor, my youngest dog.">
+            <img src="images/kobe.jpg" alt="Kobe, my grandmother's cat">
+            <img src="images/sky.jpg" alt="Sky, my oldest dog.">
 
             <h3 class="gallery-subheaders">Other Pictures</h3>
-            <img src="images/self.jpg" alt="">
-            <img src="images/family.jpg" alt="">
+            <img src="images/self.jpg" alt="Claudia on the beach">
+            <img src="images/family.jpg" alt="Claudia an her family in New York.">
```
Comment (sebagabs) [inline portfolio/src/main/webapp/gallery.html]: I like this :)

**Pull request merged**